server/services/diplomacy.js

```diff
@@ -38,7 +38,7 @@ module.exports = class DiplomacyService {
 
         let isAllied = statusTo === 'allies' && statusFrom === 'allies';
```

Comment:
This needs to be true if both player IDs are equal.

Reply:
Same with `statusTo` and `statusFrom` above. They need to be `allies` if player IDs are the same.

Reply:
Is checked one row lower

Reply:
You should be able to do the same player id check in the `statusTo` and `statusFrom` assignments, then the rest of the logic will take care of itself.

Reply:
Actualy if there the same why check the rest?